### ISSUE TITLE
Example for modifying on...Clicked behaviours within FragmentActivity (specifically: remove on...LongClicked behaviours)?

### ISSUE THREAD

**Issue by 1sm3t** [mikepenz/AboutLibraries#280]
First of all: Big sorry for this very most likely really unnecessary question. 

To make it as short as possible: How would I go about modifying the behaviours of the various onClicked events (onIconClicked, onContentLongClicked, etc. ) or even enabling them for that matter? Within the FragmentActivity libsListener is providing these events, but I can't get them to work (I am not sure what view and library values I am supposed to  provide) and even if I managed to do so, I wouldn't know what to do next If I wanted to try to remove all of the onLongClicked actions.. 

Again, sorry for this question. An example of removing/ modifying the default on...Clicked behaviours within the FragmentActivity would be greatly appreciated. I already tried to dig through all of the information I could find on this git, but even when importing the entire sample + library the onClicked listeners are not enabled by default. Additionally, I am unable to import the library into my project as a module, because gradle keeps driving me insane, thus I am unable to remove the onLongClicked behaviours within the LibraryItem class directly. 

Any kind of response would be greatly appreciated.

**Comment by mikepenz** [comment]
@1sm3t there is no solution to remove the click listener at all. You can prevent the default behaviour though. 

So for this case you just use the `LibsConfiguration` and define the `Listener` then you implement all methods and return `true` (which means the event was consumed)

https://github.com/mikepenz/AboutLibraries/blob/develop/library/src/main/java/com/mikepenz/aboutlibraries/ui/item/LibraryItem.java#L136

**Comment by 1sm3t** [comment]
@mikepenz Thank you very much for your quick reply. It eventually helped me realize how stupid I was. I actually already had created the libsListener (basically I took over all the code from your FragmentActivity), however, I removed the part where the Drawer is built. That in turn deleted the assignment of the libsListener and thus the methods never fired for me. Adding ".withListener(libsListener)" to the "LibsSupportFragment fragment = new LibsBuilder()" resolved the issue I experienced. 

Again, thank you very much!
